base_layer/wallet_ffi/src/lib.rs

```diff
@@ -5862,6 +5940,21 @@ mod test {
                 .runtime
                 .block_on((*bob_wallet).wallet.db.set_chain_meta(meta_data.clone()))
                 .unwrap();
+
+            // number of confirmations
+            let num_confirmations_required = wallet_get_num_confirmations_required(alice_wallet, error_ptr);
+            assert_eq!(num_confirmations_required, NUM_CONFIRMATIONS_REQUIRED_DEFAULT);
```

Comment:
So here I would use `TransactionServiceConfig::default().num_confirmations_required`

Reply:
I agree that is neater 👌 will do